modules/ossm-control-plane-web.adoc

```diff
@@ -10,7 +10,7 @@ You can deploy a basic `ServiceMeshControlPlane` by using the web console.  In t
 .Prerequisites
 
 * The {ProductName} Operator must be installed.
-* An account with the `cluster-admin` role.
+* An account with the `cluster-admin` role. If you use Red Hat OpenShift Dedicated, you must have an account with the `dedicated-admin` role.
```

Comment:
Now that I'm looking at this in a preview, I wonder if it wouldn't make sense to add this to the first step instead of in the prerequisites, the way you did for the CLI topic?  What do you think?

Reply:
Interesting question. On the surface, I think it might be beneficial to
present the content the same way - either in the prereq or in the step.

In general, do you think it would be helpful to customers if we explained
what account and role was needed in the prereq before the customer attempts
to start the task? In this particular instance, the information is in step
1 so it's near the beginning of the topic, but what if it were a few steps
further into the task? Would it have been helpful to know by reading the
prereqs before you began the task?